packages/edit-post/src/plugins/index.js

```diff
@@ -38,6 +39,7 @@ registerPlugin( 'edit-post', {
 							<CopyContentMenuItem />
 							<MenuItem
 								role="menuitem"
+								icon={ external }
```

Comment:
Thanks so much for picking this up 👍 

In addition to the icon, the other thing that's usually present on these external links is some visually hidden text for screenreader users.

I think you should be able to follow how the External Link component does that for implementing it on these two MenuItems:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/external-link/index.js#L38-L43

Alternatively, as @jasmussen proposes a prop for the MenuItem component could be an option so that it adds the icon and the visually hidden text implicitly.

Reply:
Thanks for the hint, i didn't knew about that. I have added the visually hidden text like in the example, works well.

Reply:
I'll refresh my PRs and rebase them off whatever lands here. In the mean time, this PR should not be held up. 

This is what the preview dropdown looks like:

<img width="327" alt="Screenshot 2020-08-24 at 09 50 02" src="https://user-images.githubusercontent.com/1204802/91018080-686ade00-e5ef-11ea-9b84-66c07b3f1791.png">

The same feedback applies to that, it looks a little weird on the left. But because this PR looks the same, I say let's move this forward. 👍 👍 from a design POV and thank you for the PR! If this gets a quick sanity check, we should land it.